setup.py

```diff
@@ -3,9 +3,9 @@
 
 
 def supports_av():
-    h = Popen("ldconfig -p | grep toxav", shell=True, stdout=PIPE)
+    h = Popen("ld -ltoxav", shell=True, stderr=PIPE)
     out, err = h.communicate()
-    return 'toxav' in str(out)
+    return 'toxav' not in str(err)
```

Comment:
since ld returns `ld: cannot find -ltoxxav`, you should be more specific when checking the result.
e.g. `re.search(r'cannot find.*toxav.*') is not None`